themes/cardinal_service/lib/js/cardinal_service.js

```diff
@@ -17,7 +17,13 @@
         $("html, body").animate({scrollTop: 0}, "slow");
       });
 
+      if($('body').is('.section-opportunities')){
+        $('.node-su-opportunity-su-opp-dimension').siblings('.node-su-opportunity-su-opp-type').before('<span>, </span>');
+        $('.node-su-opportunity-su-opp-commitment').siblings('.node-su-opportunity-su-opp-service-theme').before('<span> in </span>');
+      }
```

Comment:
I think you need spaces after the `if` and before the `{`. Also whenever doing selections, you should use the `context`. see line 15 for example. basically it prevents adding the markup twice if ajax is ran.

Reply:
or you can also use `$('.something').once('my-identifier').doThisStuff` which will run the `doThisStuff` only one time